Omnibus.md

```diff
@@ -3126,4 +3126,303 @@ Zombie Walk (1st Circle)
+must meet in order to vote there. You must have hit at least six events,
+including your own, in the same year that you held your event. Additionally
+only one EH per day of the event will be eligible to vote. For the purposes
+of this rule, a day runs from sunrise that calendar day to sunrise the next
+calendar day, and an event day must be at least six hours long. For example,
+an event starting at 9:00 pm on Friday and ending more than six hours after
+sunrise on Sunday may have three EHs.
+
+If an EH cannot or does not attend their own event (due to real life
+emergencies, extenuating circumstances, etc.), they may still attend the
+EH meeting, but their eligibility to vote and back magic items will be
+reviewed by the remaining members of the Event Holders' Council to determine
+their eligibility on a case-by-case basis.
+
+No proxies will be accepted at the Event Holders' Council. Voting at the
+Event Holders’ Council will be by majority rule - a two-thirds majority of yes
+over no, with abstentions not being counted, equals a majority. No guests
+shall be permitted to attend the Event Holders’ Council. The Moderator,
+Co-Moderator, and Secretary will be considered staff and not guests. Expert
+witnesses may be allowed if necessary, with an appropriate vote.
+
+Proposals that are supported by at least 10 EHs, who are legally able to
+attend and vote at the Event Holders' Council, will bypass the need for
+approval at the Players' Meeting. The proposal and indications of EH support
+must be submitted to the organizer(s) of the Event Holders’ Council by the
+EH-supported proposal submission deadline, which is one week prior to the
+Players' Meeting. Supported proposals must also contain the exact wording to
+be added, removed, or modified within the Omnibus, and include the section to
+be modified. Proposals created or submitted after this deadline will need to
+be approved at the Players' Meeting as detailed in the section above.
+
+Note: A brief summary of the effects of the proposed change should ideally be
+included as part of longer or more complex proposals; it is not a requirement,
+but a simple bulleted list can help both players and EHs understand the impact
+the proposal would have, reducing confusion and speeding up the meetings.
+
+Each year at the Event Holders' Council several marshals are elected to help
+streamline the administration involved with running the Realms. Below you will
+find the titles, names, contact information, and a brief description of their
+duties:
+
+**Death Marshal:** Keith Cronyn (<Keith.Cronyn@gmail.com>, 603-819-8689). Tracks
+and maintains ticks that are reported by EHs from their event for a character,
+or reported by a character's player for their own character.
+
+**Event List Administrator:** Ian Pushee (<bright@giantsquid.net>). Responsible
+for maintaining an official list of events. The list is available online at
+http://www.realmsnet.net.
```

Comment:
This can also be a link.  I think I consistently use `[url](url)` and `<e-mail@address>` for things where I want the link text to match the link address?